index.html

```diff
@@ -300,6 +300,15 @@ <h2 style="margin-top:-20px;">Getting Involved: Hardware WG</h2>
 			<li>Mission is to reduce barriers to entry for using and developing hardware libraries in Node.js.</li>
 		</ul>
 	</div></section>
+	
+	<section class="slide"><div>
+		<h2 style="margin-top:-20px;">Getting Involved: Tracing WG</h2>
+		<ul>
+			<li>Mission is to provide a portable, high performance tracing API for Node.js.</li>
```

Comment:
@Qard 

How does this sound?

old:

* Mission is to provide a portable, high performance tracing API for Node.js.

new:

* Mission is to provide portable tools and hooks to access high performance tracing capabilities in Node.js.